hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/HapiUtils.java

```diff
@@ -97,15 +97,6 @@ public class HapiUtils {
     public static final Comparator<Timestamp> TIMESTAMP_COMPARATOR =
             Comparator.comparingLong(Timestamp::seconds).thenComparingInt(Timestamp::nanos);
 
-    /** A {@link Comparator} for {@link SemanticVersion}s that ignores
```

Comment:
This code migrated to another `HapiUtils` class residing in `platform-sdk` - `com.swirlds.platform.state.spi.HapiUtils`